docs/proposals/probabilistic_proofs.md

```diff
@@ -0,0 +1,241 @@
+- How many failures until N success?
+- How many success until N failures?
+- How much scalability does the network need?
+- With what likelihood should an attacker get away?
+-->
+
+### Example
+
+Assume:
+
+- `ProofRequirementThreshold = 100 POKT`
+- `ProofRequestProbability = 0.25`
+
+If the `Claim` is greater than or equal to `100 POKT`, a proof is mandatory and the model is _"short-circuited"_. Therefore, the attacker can _freeload_ by submitting claims for `99.99 POKT` and hope they never get caught. If they do get caught (i.e. `Success`), the burn (i.e. `ProofMissingPenalty`) should exceed the total reward accumulated.
+
+Since each claim is independent, an attacker would never submit a `Claim` exceeding `ProofRequirementThreshold`, and therefore have a `ProofRequestProbability` likelihood of being required to submit a proof.
+
+### Model
+
+A [Geometric_distribution](https://en.wikipedia.org/wiki/Geometric_distribution) is used to identify the probability of `k` failures (sample space containing an attacker getting away) until a single success (an attacker is caught).
+
+$$ p = ProofRequestProbability $$
+
+$$ q = 1 - p $$
+
+$$ Pr(X=k) = (1-p)^{k-1}p $$
+
+$$ k = \frac{ln(\frac{Pr(X=k)}{p})}{ln(1-p)} + 1 $$
+
+![download](https://user-images.githubusercontent.com/1892194/220803154-90dcdd6b-8141-40d2-9cca-ed27a995fcfb.png)
+
+### Selecting Values
+
+#### Calculation
+
+_TODO(olshansky): Look at claims data to figure out the value for ProofRequirementThreshold_
+
+`ProofRequirementThreshold` should be as small as possible so that most such that most Claims for into the probabilistic bucket, while also balancing out penalties that may be too large for faulty honest Servicers. It will be selected to be `2σ` above the Claim `μ` such that `97.3%` fall into the `ProofRequestProbability` part of the piecewise function.
+
+`ProofRequestProbability (p)` is selected as `0.25` to enable scaling the network by `4x`.
+
+`Pr(X=k) > 0` must be greater than 0 while minimizing `k` as much as possible, so it is selected at approximately `0.01.`.
+
+$$ k = \frac{ln(\frac{Pr(X=k)}{p})}{ln(1-p)} + 1 $$
+
+$$ k = \frac{ln(\frac{0.01}{0.25})}{ln(1-0.25)} + 1 $$
+
+$$ k ≈ 12.19 $$
+
+Selecting `k = 12` implies that there is an `~1%` of 12 failures (includes the sample space where an attacker gets away) until a single success (an attacker is caught). To deter this behaviour, the `ProofMissingPenalty` should be selected as 12 times `ProofRequirementThreshold`.
+
+To answer this question, we need to:
+
+- Pick `p` - variable for scaling the network (e.g. 0.25 => 4x network scale)
+- Select `Pr(X=k)` - the likelihood of `k` failures
+- Introduce `BurnForFailedClaimSubmission` - penalty for getting caught in falsifying a claim (or failing to submit a real claim)
+
+`ProofRequirementThreshold` - Should be set to a value whereby that that is `2σ` greater than the mean claim. This will mean that the other `97.3%` of claims will be subject to `ProofRequestProbability`.
```

Comment:
You are supposing they have a normal distribution, lets hope that's the case, anyway we will know once I give you the data.

Reply:
That was definitely an assumption I was making. I saw your comment in #1523 and will follow up there.